src/force-latest-compat.jl

```diff
@@ -0,0 +1,47 @@
+using TOML
+
+@inline function latest_compat(range::VersionRange)
```

Comment:
Random question but why is everything here inlined?

Reply:
Not sure; I'm sure it made sense in my head when I wrote it.

Fixed now.